azkaban-common/src/main/java/azkaban/storage/DatabaseStorage.java

```diff
@@ -64,7 +66,23 @@ public String put(final StorageMetadata metadata, final File localFile) {
   }
 
   @Override
-  public boolean delete(final String key) {
+  public InputStream getDependency(final Dependency dep) {
+    throw new UnsupportedOperationException(
+        "Not implemented yet. Must use HdfsStorage or LocalStorage.");
+  }
+
+  @Override
+  public boolean dependencyFetchingEnabled() {
+    return false;
+  }
+
+  @Override
+  public String getDependencyRootPath() {
+    return null;
+  }
```

Comment:
can you add some comments about these two methods? if they're not being used, you can just say so.